docs/pages/admin-guides/access-controls/access-lists/nested-access-lists.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Nested Access Lists
+description: Learn how to use nested Access Lists to manage complex permissions and grant inheritance in Teleport.
+---
+
+This guide will help you:
+
+- Understand how nesting and inheritance work in Access Lists
+- Create a nested Access List
+- Verify inherited permissions granted through the nested Access List
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- A user with the `editor` role or equivalent permissions to create and manage Access Lists.
```

Comment:
For organizations looking to craft least-privileged roles, it might make sense to list the specific role permissions.